_rules/document-has-no-repeated-content-g0m9ub.md

```diff
@@ -166,7 +166,7 @@ In this document, there is a [block of repeated content][] before the [main bloc
 
 #### Inapplicable Example 1
 
-This [document][] is not an [HTML web page][].
+This [document][] is not a [HTML web page][].
```

Comment:
@ShadowBB and @Jym77: Wait a second, isn't the common pronunciation of the letter H _aitch_, not _haitch_? That's what Wikipedia suggests at least: https://en.wikipedia.org/wiki/English_alphabet#Letter_names. In that case "an" is correct.

Reply:
Huh. People seem somewhat divided:
https://english.stackexchange.com/questions/25742/is-it-supposed-to-be-a-html-or-an-html

"a HTML page" seems to have more results on google, but not by much. I think both are okay and we should pick the one that we have used on the website before and that seems to be "an html (element/page)". So I retract my earlier request to use "a html".

Reply:
Every single answer on that SE question states that "an HTML" is correct though, for the exact same reason I'm outlining 🙈